### PR TITLE
localhost styling

### PR DESCRIPTION
**Pull request by Raruto**
I took the svg file from the [octicon-code](https://octicons.github.com/icon/code/) icon.

However, we can also achieve a similar effect by replacing the last rule with the following:

```css
#wpadminbar #wp-admin-bar-site-name > .ab-item::before {
  content: "\f475" !important;
  font: normal 17px/1.25 dashicons;
  color: #ff0;				
}
```

I prefer the **octicon version** because it has slightly more space in the middle (**best for small sizes**)

### CODE REVIEW

**Comment by afragen** [comment]
Can you post a screenshot of what this does?

File: src/Local_Development/Extras.php [hunk 1]
```diff
@@ -105,6 +105,21 @@ public function page_init() {
 				]
 			);
 		}
+
+		if( in_array( $_SERVER['REMOTE_ADDR'], [ '127.0.0.1', '::1' ] ) ) {
```
Comment (afragen) [inline src/Local_Development/Extras.php]: My local environment updates my hosts file, so localdev.test opens. How does this change work with something like that?

**Comment by Raruto** [comment]
local-development

**Comment by afragen** [comment]
Should I ask about the color choice as well as the a11y contrast ratio?

**Comment by afragen** [comment]
Contrast ratio is 4.78, excellent. Contrast ratio is 5.13 if we use white text. I think that might be preferable to having 2 different text colors present.

File: src/Local_Development/Extras.php (hunk 1 above)
Comment (afragen) [inline src/Local_Development/Extras.php]: in_array needs third parameter `true` for WPCS

**Comment by Raruto** [comment]
> Should I ask about the color choice as well as the a11y contrast ratio?

Well, for the moment I would not go too far ..

> Contrast ratio is 4.78, excellent. Contrast ratio is 5.13 if we use white text. I think that might be preferable to having 2 different text colors present.

I slightly increased the font weight: https://webaim.org/resources/contrastchecker/?fcolor=FFFF00&bcolor=008000 (**yellow on green is more noticeable**)

File: src/Local_Development/Extras.php (hunk 1 above)
Comment (Raruto) [reply src/Local_Development/Extras.php]: Ok thanks, unfortunately my local php config is temporarily KO ...

**Comment by Raruto** [comment]
I did some further tests with bold text and it seems too much aggressive / intrusive.
If that's okay I'd keep it that way..

**Comment by afragen** [comment]
@Raruto what is your WordPress.org username so I can add you to readme.txt?

**Pull request merged**